cgmanifest.json

```diff
@@ -275,7 +275,7 @@
 				"threshold for this to be true is not precisely defined by law.",
 				"  If such an object file uses only numerical parameters, data",
 				"structure layouts and accessors, and small macros and small inline",
-				"functions (ten lines or less in length), then the use of the object",
+				"functions (ten lines or fewer in length), then the use of the object",
```

Comment:
We cannot change the license. It's not something we own, and take verbatim

Reply:
lol